.README/rules/mark-used.md

```diff
@@ -0,0 +1,27 @@
+# `mark-used`
```

Comment:
I would some feedback on the name. It doesn't convey that this has to do with "types" being marked as used. An alternative would be `used-types` as a tribute to the `valid-types` rule, which already exists in the package.

Reply:
`used-types` sounds good to me

Reply:
Pushed a rename to "used-types".